test/e2e/scenario/recorder/viewports.scenario.ts

```diff
@@ -34,7 +35,9 @@ describe('recorder', () => {
         await performSignificantZoom()
 
         await browserExecute(() => {
-          window.dispatchEvent(new Event('resize'))
+          const event = new Event('resize') as TrustableEvent
+          event.__ddIsTrusted = true
+          window.dispatchEvent(event)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏any idea why this one need to be trusted manually?

Reply:
Ho right. setupTrustedEvents() should be called before the header: https://github.com/DataDog/browser-sdk/blob/6562f01af1f3aa2929e7d795945fbb2eddc09eba/test/e2e/lib/framework/pageSetups.ts#L165-L166
Then no need to do `event.__ddIsTrusted = true` here